examples/complex-chart/app.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
I'm gonna error on the side of caution and say that yes it must be in every file